app/models/game-history.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Module dependencies.
+ */
+/* eslint-disable no-unused-vars */
+const mongoose = require('mongoose'),
+  config = require('../../config/config'),
+  Schema = mongoose.Schema;
+
+/**
+ * Answer Schema
+ */
+const GameHistorySchema = new Schema({
+  gameID: { type: String, required: true },
+  started: { type: Date, default: Date.now },
+  ended: { type: Boolean, default: false },
+  rounds: { type: Number, default: 0 },
+  creator: { type: String, default: '', required: true },
```

Comment:
The `default` is no longer needed since this is a required field. Sorry, I didn't mention that before now.